cc/mcts_player.cc

```diff
@@ -14,6 +14,7 @@
 
 #include "cc/mcts_player.h"
 
+#include <algorithm>
```

Comment:
eh?

Reply:
actually, now that i'm here i notice i didn't see anything that takes care of ignoring the padded-out features

Reply:
algorithm is required for std::max.

the padded-out features are ignored because InferenceServer::PutOutputs only iterates over the items in the *requested* batch, not the actual batch sent to the model. See: https://github.com/tensorflow/minigo/pull/256/files#diff-5ff231d14f937b5c55b58e7a87281ee1R149

Reply:
nice, got it